fhir-parent/pom.xml

```diff
@@ -10,11 +10,11 @@
     <name>IBM FHIR Server</name>
 
     <properties>
-        <liberty.version>20.0.0.9</liberty.version>
+        <liberty.version>20.0.0.12</liberty.version>
         <derby.version>10.14.2.0</derby.version>
-        <db2.version>11.5.0.0</db2.version>
-        <postgresql.version>42.2.12</postgresql.version>
-        <cxf.version>3.3.6</cxf.version>
+        <db2.version>11.5.5.0</db2.version>
+        <postgresql.version>42.2.18</postgresql.version>
+        <cxf.version>3.3.9</cxf.version>
```

Comment:
Should we go up to 3.4.2?

Reply:
We probably could if we want to.  I stayed on the more catious side since it looked like most of the important fixes were backported to 3.3.x anyway.  I still want to see if we can remove the embedded CXF version altogether since we should be able to get everything we need from Liberty.

Reply:
maybe we should have an issue to track that work? and target 4.6.0?

Reply:
https://github.com/IBM/FHIR/issues/1179 ;-)